treeherder/etl/classification_mirroring.py

```diff
@@ -27,25 +26,16 @@ def generate_request_body(self):
         """
         Create the data structure that will be sent to Elasticsearch.
         """
-        with JobsModel(self.project) as jobs_model, ArtifactsModel(self.project) as artifacts_model:
+        with JobsModel(self.project) as jobs_model:
             job_data = jobs_model.get_job(self.job_id)[0]
             buildtype = " ".join(sorted(OptionCollection.objects.values_list(
                 'option__name', flat=True).filter(
                     option_collection_hash=job_data["option_collection_hash"])))
             revision_list = jobs_model.get_resultset_revisions_list(job_data["result_set_id"])
-            buildapi_artifact = artifacts_model.get_job_artifact_list(0, 1, {
-                'job_id': set([("=", self.job_id)]),
-                'name': set([("=", "buildapi")])
-            })
-            if buildapi_artifact:
-                buildname = buildapi_artifact[0]["blob"]["buildername"]
-            else:
-                # OrangeFactor needs a buildname to be set or it skips the failure
-                # classification, so we make one up for non-buildbot jobs.
-                buildname = 'non-buildbot %s test %s' % (job_data["platform"], job_data["job_type_name"])
+            ref_data_name = job_data["ref_data_name"]
```

Comment:
I wasn't meaning the taskcluster case, I mean the buildbot one.

I'd skim read the test in test_classification_mirroring.py as one for a buildbot job, and so:
```
"buildname": "39643b5073cfb9473042884bfd3ced0289b3d7dd"
```
...is what would get sent to OrangeFactor in the buildbot case.

If that's not true, then that's fine.

Though I'm so over re-wallpapering OrangeFactor and related that I'm happy for you to just land this regardless tbh..